src/util/TestCaseExcelExportUtil.ts

```diff
@@ -129,23 +129,22 @@ export const createExcelExportDtosForAllTestCases = (
     const testCaseExecutionResultDtos: TestCaseExecutionResultDto[] = [];
     const testCasesByGroup: TestCase[] = getTestCasesByGroup(measure, group.id);
     testCasesByGroup?.forEach((currentTestCase) => {
-      const groupPopulations = currentTestCase.groupPopulations?.filter(
+      const groupPopulation = currentTestCase.groupPopulations?.filter(
```

Comment:
We could replace filter with find or use lodash to perform the same

Reply:
@RohitKandimalla i tried to use find, but the data returned empty, so I changed it back to filter.